css/main.css

```diff
@@ -586,38 +593,38 @@ div.track-label:hover {
     opacity: 1.0;
 }
 
-div.tracklist-label {
+.jbrowse div.tracklist-label {
     padding: 3px;
     background-color: #ddd;
     background-color: rgba( 220, 220, 220, 0.9 );
 }
-div.tracklist-label:hover {
+.jbrowse div.tracklist-label:hover {
     background-color:  #396494;
     border: 1px solid black;
 }
 
-div.tracklist-container.dojoDndItemSelected {
+.jbrowse div.tracklist-container.dojoDndItemSelected {
     background-color:  #396494;
     background-color: rgb(110, 132, 156)
 }
-div.tracklist-container.dojoDndItemSelected .tracklist-label {
+.jbrowse div.tracklist-container.dojoDndItemSelected .tracklist-label {
     background-color: #b2c3d6;
 }
 
-.jbrowseSimpleTrackSelector {
+.jbrowse .jbrowseSimpleTrackSelector {
     background: #f2f2f2;
     background-image: url('../img/tracklist_bg.png');
     background-repeat: repeat-y;
     background-position: right;
 }
-.jbrowseSimpleTrackSelector h2 {
+.jbrowse .jbrowseSimpleTrackSelector h2 {
     margin: 0.5em 0 0 0.5em;
 }
 
-div.textfilter {
+.jbrowse div.textfilter {
     margin: 0.5em 0 1em 0.3em;
 }
-div.textfilter input {
+.jbrowse div.textfilter input {
     padding: 2px 0 2px 18px;
     width: 100%;
     height: 100%;
```

Comment:
It looks like this "tall filter tracks" thing would happen on master branch too if you used jquery load(). That would be interesting to fix I suppose, but isn't specific to this branch